### PR TITLE
en, zh: add release notes for v1.1.13

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

add release notes for v1.1.13

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: en/releases/release-1.1.13.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# TiDB Operator 1.1.13 Release Notes
+
+Release date: July 2, 2021
+
+TiDB Operator version: 1.1.13
+
+## Improvements
+
+- TiCDC supports setting TLS certs for sinks ([#3926](https://github.com/pingcap/tidb-operator/pull/3926), [@handlerww](https://github.com/handlerww))
```
Comment (qiancai) [inline en/releases/release-1.1.13.md:13]:
```suggestion
- TiCDC supports setting TLS certificates for sinks ([#3926](https://github.com/pingcap/tidb-operator/pull/3926), [@handlerww](https://github.com/handlerww))
```

File: en/releases/release-1.1.13.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Support using the TiKV version as the tag for BR `toolImage` if no tag is contained ([#4048](https://github.com/pingcap/tidb-operator/pull/4048), [@KanShiori](https://github.com/KanShiori))
```
Comment (qiancai) [inline en/releases/release-1.1.13.md:14]:
```suggestion
- Support using the TiKV version as the tag for BR `toolImage` if no tag is specified ([#4048](https://github.com/pingcap/tidb-operator/pull/4048), [@KanShiori](https://github.com/KanShiori))
```

File: en/releases/release-1.1.13.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- Support handling PVC during scaling of TiDB ([#4033](https://github.com/pingcap/tidb-operator/pull/4033), [@csuzhangxc](https://github.com/csuzhangxc))
+- Masks the backup password in logging ([#3979](https://github.com/pingcap/tidb-operator/pull/3979), [@dveeden](https://github.com/dveeden))
```
Comment (qiancai) [inline en/releases/release-1.1.13.md:16]:
```suggestion
- Support masking the backup password in logging ([#3979](https://github.com/pingcap/tidb-operator/pull/3979), [@dveeden](https://github.com/dveeden))
```

File: en/releases/release-1.1.13.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Bug Fixes
+
+- Fix the issue that TiDB Operator may panic when heterogeneous clusters are deployed ([#4054](https://github.com/pingcap/tidb-operator/pull/4054) [#3965](https://github.com/pingcap/tidb-operator/pull/3965), [@KanShiori](https://github.com/KanShiori) [@july2993](https://github.com/july2993))
```
Comment (qiancai) [inline en/releases/release-1.1.13.md:20]:
```suggestion
- Fix the issue that TiDB Operator may panic during the deployment of heterogeneous clusters ([#4054](https://github.com/pingcap/tidb-operator/pull/4054) [#3965](https://github.com/pingcap/tidb-operator/pull/3965), [@KanShiori](https://github.com/KanShiori) [@july2993](https://github.com/july2993))
```

File: en/releases/release-1.1.13.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+- Fix the issue that TiDB instances still exist in TiDB Dashboard after scaled in ([#3929](https://github.com/pingcap/tidb-operator/pull/3929), [@july2993](https://github.com/july2993))
```
Comment (qiancai) [inline en/releases/release-1.1.13.md:21]:
```suggestion
- Fix the issue that TiDB instances are kept in TiDB Dashboard after being scaled in ([#3929](https://github.com/pingcap/tidb-operator/pull/3929), [@july2993](https://github.com/july2993))
```

File: zh/releases/release-1.1.13.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+发布日期：2021 年 7 月 2 日
+
+TiDB Operator 版本：1.1.13
+
+## 优化提升
+
+- TiCDC 支持设置连接下游的 TLS 证书 ([#3926](https://github.com/pingcap/tidb-operator/pull/3926), [@handlerww](https://github.com/handlerww))
+- 支持在未为 BR `toolImage` 指定 tag 时将 TiKV 版本作为 tag ([#4048](https://github.com/pingcap/tidb-operator/pull/4048), [@KanShiori](https://github.com/KanShiori))
+- 支持在扩缩容 TiDB 过程中协调 PVC ([#4033](https://github.com/pingcap/tidb-operator/pull/4033), [@csuzhangxc](https://github.com/csuzhangxc))
+- 支持在日志中隐藏执行备份操作时的密码 ([#3979](https://github.com/pingcap/tidb-operator/pull/3979), [@dveeden](https://github.com/dveeden))
```
Comment (qiancai) [inline zh/releases/release-1.1.13.md:16]:
```suggestion
- 支持在 backup 日志中隐藏对数据库密码的展示 ([#3979](https://github.com/pingcap/tidb-operator/pull/3979), [@dveeden](https://github.com/dveeden))
```

Comment (qiancai) [reply zh/releases/release-1.1.13.md:16]: 同一个 issue, 在不同版本 release note 中建议保持一致
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/release-1.2.0-rc.1

File: zh/releases/release-1.1.13.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+# TiDB Operator 1.1.13 Release Notes
+
+发布日期：2021 年 7 月 2 日
+
+TiDB Operator 版本：1.1.13
+
+## 优化提升
+
+- TiCDC 支持设置连接下游的 TLS 证书 ([#3926](https://github.com/pingcap/tidb-operator/pull/3926), [@handlerww](https://github.com/handlerww))
```
Comment (qiancai) [inline zh/releases/release-1.1.13.md:13]: 这条 TiDB Operator 改了啥呢？看起来是 TiCDC 的优化提升呢

Comment (csuzhangxc) [reply zh/releases/release-1.1.13.md:13]: 是 TiDB Operator 支持给 TiCDC 配置到下游的 TLS 证书。有什么好的修改意见？

Comment (qiancai) [reply zh/releases/release-1.1.13.md:13]:
```suggestion
- 支持为 TiCDC 配置到下游的 TLS 证书 ([#3926](https://github.com/pingcap/tidb-operator/pull/3926), [@handlerww](https://github.com/handlerww))
```

File: en/releases/release-1.1.13.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+# TiDB Operator 1.1.13 Release Notes
+
+Release date: July 2, 2021
+
+TiDB Operator version: 1.1.13
+
+## Improvements
+
+- TiCDC supports setting TLS certificates for sinks ([#3926](https://github.com/pingcap/tidb-operator/pull/3926), [@handlerww](https://github.com/handlerww))
```
Comment (qiancai) [inline en/releases/release-1.1.13.md:13]:
```suggestion
- Supports configuring TLS certificates for TiCDC sinks ([#3926](https://github.com/pingcap/tidb-operator/pull/3926), [@handlerww](https://github.com/handlerww))
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**